scripts/changelog/changelog.go

```diff
@@ -85,7 +85,7 @@ func main() {
 		for scanner.Scan() {
 			line := scanner.Text()
 			if strings.Contains(line, "%changelog") {
-				amazonLinuxSpecBase += fmt.Sprintln("%changelog")
+				amazonLinuxSpecBase += "%changelog"
```

Comment:
with removal of Sprintln, should this be "%changelog\n" ?

Reply:
yes I think you may be correct

Reply:
i can fix this one